raiden-dapp/src/components/RaidenDialog.vue

```diff
@@ -29,9 +29,9 @@ import { Component, Emit, Vue, Prop } from 'vue-property-decorator';
 
 @Component({ components: { BlurredOverlay } })
 export default class RaidenDialog extends Vue {
-  @Prop({ required: true, default: false })
+  @Prop({ required: true, default: false, type: Boolean })
   visible!: boolean;
-  @Prop({ required: false })
+  @Prop({ required: false, default: false, type: Boolean })
```

Comment:
Why were these type defs needed again? For eslint to find missing required props?

Reply:
So the default is there because otherwise, the property will be undefined which makes vue complain and might or might not create issues with reactivity. Adding type especially for booleans allows you to use the existence/non-existence of the attribute as boolean. 

So instead of having to do `:hide-close="true"` you can just use `hide-close` and this will be interpreted the same as the first one. If you don't have the type there the second option won't work.